Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```
@@ -867,6 +867,7 @@
   metamorphicChangeColor: false
   metabolisms:
     Drink:
+      metabolismRate: 0.5
```

```suggestion
```
Isn't this the default value? So it can be removed I think.
```suggestion
      metabolismRate: 0.5
```